### PR TITLE
Dividing intermediate personas in three

### PR DESCRIPTION
**Pull request by gvwilson**
1. Create a new persona who is a mathematical/statistical expert.
2. Distinguish clearly from personas for domain expertise and coding knowledge.
3. Reorder from least to most experienced.
4. Change usage examples to match.
5. Make it clearer that instructors can translate these personas as needed, but should only create new ones if absolutely necessary.

### CODE REVIEW

File: docs/courses/design/personas.md [hunk 1]
```diff
@@ -15,60 +15,51 @@ e.g.,
...
 
-- Head quantitative analyst in small hedge fund.
-
-**Career Goal**
-
-After developing his skill set and finding out what he likes in
-previous jobs, Alex is set on staying in this role for quite a
-while. He was recently made part of the management team. His main
-concern is driving the company forward and outpacing competitors.
+- Financial analyst at KPMG.
```
Comment (richierocks) [inline docs/courses/design/personas.md]: Doesn't this count as domain expertise? I'd be more tempted to make Umberto a recent graduate, or give him a totally non-technical background. Perhaps he was a DJ until his hearing problems caused him to consider a new career.

File: docs/courses/design/personas.md [hunk 2]
```diff
@@ -83,27 +74,27 @@ for him lies in the apply fast way the learning experience is set up.
...
-SIndhu came out of university with knowledge about the most important
-statistical concepts and techniques. However, needing to visualise a
-lot of her findings, Sindhu quickly found that nor Excel and JMP - the
-tools used in her team - nor SPSS - the software used in university -
-were as customizable as she would have liked. Moreover, she lost a lot
-of time building her model from scratch each time.  A quick survey
-among research scientist friends led her to R and DataCamp.  After two
-free courses and the relief that it was actually doable, Sindhu
-subscribed.
+SIndhu came out of university with expert-level knowledge of
```
Comment (richierocks) [inline docs/courses/design/personas.md]: bad cap: "SIndhu"

File: docs/courses/design/personas.md [hunk 3]
```diff
@@ -181,48 +172,113 @@ her tinker with code as she went along.
...
+- Personal: Interested in teaching courses himself some day.
+
+**Previous Jobs**
+
+- Pricing analyst for power distributor.
+- Financial analyst at Bank of America.
+
+**Current Job**
+
+- Head quantitative analyst in small hedge fund.
```
Comment (richierocks) [inline docs/courses/design/personas.md]: grammar: "in a"

File: docs/courses/design/personas.md [hunk 4]
```diff
@@ -181,48 +172,113 @@ her tinker with code as she went along.
...
+## Advanced Alex
+
+- Age: 34
+- Location: Reno, Nevada, USA
+- Education: Bachelor in Mathematics and Master in Finance & Risk Management - University of Nevada
+- Personal: Interested in teaching courses himself some day.
+
+**Previous Jobs**
+
+- Pricing analyst for power distributor.
```
Comment (richierocks) [inline docs/courses/design/personas.md]: grammar: "for a"

**Comment by gvwilson** [comment]
Comments addressed - back to @richierocks

File: docs/courses/design/personas.md [hunk 5]
```diff
@@ -28,15 +28,17 @@ novice understanding of R but advanced understanding of medical imaging
...
 **Data Science Education Experience**
 
-Having only worked with excel and some SPSS in his studies, Umberto
-learned most of the modelling and analysis techniques on the job. The
-managers he worked with so far in his projects used Excel and custom
-KPMG tools, and he hasn't been in touch with data science tools as of
-now.  He thus doesn't know about them or how they could help him yet.
+Umberto's degree gave him an excellent grounding in economic theory,
+but the only quantitative work he did was with very small data sets
+using Excel and a little bit of copy-and-pase SPSS.  The managers he
```
Comment (richierocks) [inline docs/courses/design/personas.md]: typo: pase->paste

File: docs/courses/design/personas.md [hunk 6]
```diff
@@ -47,11 +49,10 @@ Umberto doesn't know DataCamp.
 
 **Career Goal**
 
-Having a not-so-common education background, Umberto is set on
-learning as much as possible working in consultancy for many different
-clients. He loves the on the job trainings and wants to build
-knowledge so he can grow into a project lead role. In the future, he
-sees himself as lead analyst for a food import/export company.
+Aware that his background hasn't given him the samel quantitative
```
Comment (richierocks) [inline docs/courses/design/personas.md]: typo: samel -> same

File: docs/courses/design/personas.md [hunk 7]
```diff
@@ -47,11 +49,10 @@ Umberto doesn't know DataCamp.
...
+skills as many of his peers, Umberto is set on learning as much as
+possible as quickly as he can. He loves the on the job trainings and
```
Comment (richierocks) [inline docs/courses/design/personas.md]: "on the job" -> "on-the-job"?

**Pull request merged**